llvm/lib/Target/BPF/AsmParser/BPFAsmParser.cpp

```diff
@@ -259,6 +261,8 @@ struct BPFOperand : public MCParsedAsmOperand {
         .Case("bswap64", true)
         .Case("goto", true)
         .Case("gotol", true)
+        .Case("gotol_or_nop", true)
```

Comment:
skeptical about this, is `gotol_or_nop` `ValidIdInMiddle`? I think we did not introduce any asm syntax like:

    if ... gotol_or_nop

Reply:
> skeptical about this, is `gotol_or_nop` `ValidIdInMiddle`? I think we did not introduce any asm syntax like:
> 
> ```
> if ... gotol_or_nop
> ```

Thanks! Will remove it later.